big_tests/test.config

```diff
@@ -18,6 +18,7 @@
 %%       so that we rein the "bag of things" approach
 {hosts, [{mim,  [{node, mongooseim@localhost},
                  {domain, <<"localhost">>},
+                 {host_type, <<"localhost">>},
```

Comment:
My idea was to use `host_type(mim)` instead - already implemented in `domain_helper`.

Reply:
Could do that, but I actually thought of using this domain_helper shortcut, Mikail did the same in his MAM PR, and I thought it'd be just faster to skip the rpc. Can do that instead anyways 🤷🏽‍♂️